bin/gltf-pipeline.js

```diff
@@ -78,6 +79,7 @@ fs.readFile(gltfPath, function (err, data) {
 
         combineMeshes(gltf);
         combinePrimitives(gltf);
+        combineNodes(gltf);
```

Comment:
Yes, that is no problem, just make sure to comment why things happen in the order that they do.